tsparser/src/parser/resources/infra/objects.rs

```diff
@@ -120,6 +120,7 @@ pub fn resolve_bucket_usage(data: &ResolveUsageData, bucket: Lrc<Bucket>) -> Opt
                 "list" => Operation::ListObjects,
                 "exists" | "attrs" => Operation::GetObjectMetadata,
                 "upload" => Operation::WriteObject,
+                "getUploadUrl" => Operation::WriteObject,
```

Comment:
Rename to `signedUploadUrl` as per above. Also, let's introduce a new `SignedUploadUrl` operation (needs to be done in legacymeta.rs as well)

Reply:
Done